include/test/battle.h

```diff
@@ -864,6 +965,11 @@ struct ItemContext
 void OpenTurn(u32 sourceLine);
 void CloseTurn(u32 sourceLine);
 void Move(u32 sourceLine, struct BattlePokemon *, struct MoveContext);
+void ExpectedMove(u32 sourceLine, struct BattlePokemon *, struct MoveContext);
+void ExpectedMoves(u32 sourceLine, struct BattlePokemon *battler, bool32 notExpected, struct FourMoves moves);
+void ExpectedSendOut(u32 sourceLine, struct BattlePokemon *battler, u32 partyIndex);
+void ExpectedSwitch(u32 sourceLine, struct BattlePokemon *battler, u32 partyIndex);
+void Score(u32 sourceLine, struct BattlePokemon *battler, u32 cmp, bool32 toValue, struct TestAiScoreStruct cmpCtx);
```

Comment:
Maybe consider calling this `ExpectScore` or `ExpectScoreCompare` or something?
```suggestion
void Score(u32 sourceLine, struct BattlePokemon *battler, u32 cmp, bool32 toValue, struct TestAIScoreStruct cmpCtx);
```